lib/libcheri_caprevoke/aarch64/libcaprevoke_md.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+static inline uint64_t
+caprev_shadow_set_fw(uint64_t * __capability fw, void * __capability user_obj,
```

Comment:
If this is purecap-only then why all the annotations and ifdefs?

Reply:
Historically, it was not purecap, and in principle on riscv64 it still builds for the hybrid world.  I guess the comment about the morello atomics has gotten lost as it was in the build hackery; I will restore it into this file.